WalletWasabi.Fluent/Views/MainWindow.axaml

```diff
@@ -20,6 +20,7 @@
         ExtendClientAreaChromeHints="Default, PreferSystemChrome, OSXThickTitleBar"
         Title="Wasabi Wallet"
         WindowState="{Binding WindowState, Mode=TwoWay}"
+        Focusable="True"
```

Comment:
This is to detect that the user has clicked outside the Flyout, in order to close it. Setting this to false will cause the Flyout to stay open because the framework considers that it should not close it, because no other element has received the focus.

This is to inform Avalonia that clicking anything in the window is considered a focus change within the window.